<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcesoDialog</class>
 <widget class="QDialog" name="ProcesoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Procesos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="num_proceso_label">
     <property name="text">
      <string>PROCESO </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="nombre_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="nombre_label">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nombre_lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="operacion_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="operacion_label">
        <property name="text">
         <string>Operaci√≥n</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="operacion_comboBox">
        <item>
         <property name="text">
          <string>Suma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Resta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Multiplicacion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Division</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="numeros_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="num1_label">
        <property name="text">
         <string>Numero 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="num1_lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="num2_label">
        <property name="text">
         <string>Numero 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="num2_lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tiempo_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="tiempo_label">
        <property name="text">
         <string>Tiempo maximo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="tiempo_spinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="id_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="id_label">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="id_lineEdit">
        <property name="cursorPosition">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProcesoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProcesoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
